stablehlo/dialect/StablehloOps.cpp

```diff
@@ -1589,42 +1623,41 @@ LogicalResult verifyConvolutionAttributes(ConvolutionOp op) {
 //  1. Input args to ConvolutionOp 'op' are RankedTypes.
 //  2. rank-of(input-type) == rank-of(output-type)
 SmallVector<int64_t> inferConvolutionOpReturnShape(
-    ConvolutionOp op, const ArrayRef<hlo::WindowDimension> window) {
-  // We keep the 'unknown' dimensions (cl/415132294) as it is in the
+    Value lhs, Value rhs, int64_t inputBatchDimension,
```

Comment:
Is there a reason there args are not put one per line?

Reply:
Fixed.